CHANGELOG.d/fix-image-sampling.md

```diff
@@ -0,0 +1,10 @@
+## Unreleased
+
+###### Gameplay
+
+###### User Interface
+- Fixed an issue causing some UI components to appear blurry.
```

Comment:
```suggestion
- Fixed an issue causing some UI components to appear distorted.
```

I don't think these artifacts could be described as "blur", as everything looks crisp, just jagged in odd ways.